fe/fe-core/src/main/java/com/starrocks/privilege/StorageVolumePEntryObject.java

```diff
@@ -114,4 +115,17 @@ public int hashCode() {
     public PEntryObject clone() {
         return new StorageVolumePEntryObject(id);
     }
+
+    @Override
```

Comment:
better add ut or sql-tester to cover the scenario.

Reply:
> better add ut or sql-tester to cover the scenario.

done